recipes/gobject-introspection/all/test_package/conanfile.py

```diff
@@ -8,11 +8,12 @@ class TestPackageConan(ConanFile):
 
     def build(self):
         if self.settings.os != 'Windows':
-            self.run('g-ir-annotation-tool --version', run_environment=True)
-            self.run('g-ir-compiler --version', run_environment=True)
-            self.run('g-ir-generate --version', run_environment=True)
-            self.run('g-ir-inspect -h', run_environment=True)
-            self.run('g-ir-scanner --version', run_environment=True)
+            with tools.environment_append({'PKG_CONFIG_PATH': "."}):
```

Comment:
This should be moved to `test` + only run when not cross building.
It's okay to test for presence of the .pc sections, but it's not okay to run the executables.

Reply:
I'm not sure about this, I think these tools are not meant to run on the host platform. They are build tools

Reply:
When executing the test, won't `g-ir-compiler` be compiled for the host machine and thus potentially not run on the build machine?
Or are these platform independent scripts?